tests/python/22_verify/22_verify.py

```diff
@@ -25,92 +15,80 @@
 from xrt_binding import *
 from ert_binding import *
 
+# found in PYTHONPATH
+import pyxrt
+
 # utils_binding.py
 sys.path.append('../')
 from utils_binding import *
 
 
 def runKernel(opt):
+    d = pyxrt.device(opt.index)
+    xbin = pyxrt.xclbin(opt.bitstreamFile)
+    uuid = d.load_xclbin(xbin)
 
-    rule = re.compile("hello*")
-    name = list(filter(lambda val: rule.match, opt.kernels))[0]
-    khandle = xrtPLKernelOpen(opt.handle, opt.xuuid, name)
+    iplist = xbin.get_ips()
 
-    grpid = xrtKernelArgGroupId(khandle, 0)
-    if grpid < 0:
-        raise RuntimeError("failed to find BO group ID: %d" % grpid)
+    rule = re.compile("hello*")
+    ip = list(filter(lambda val: rule.match(val.get_name()), iplist))[0]
+    hello = pyxrt.kernel(d, uuid, ip.get_name(), pyxrt.kernel.shared)
 
-    boHandle1 = xrtBOAlloc(opt.handle, opt.DATA_SIZE, 0, grpid)
-    buf1 = xrtBOMap(boHandle1)
-    bo1 = ctypes.cast(buf1, ctypes.POINTER(ctypes.c_char))
-    if bo1 == 0:
-        raise RuntimeError("failed to map buffer1")
-    ctypes.memset(bo1, 0, opt.DATA_SIZE)
+    boHandle1 = pyxrt.bo(d, opt.DATA_SIZE, pyxrt.bo.normal, hello.group_id(0))
```

Comment:
```suggestion
    boHandle1 = pyxrt.bo(d, opt.data_size, pyxrt.bo.normal, hello.group_id(0))
```

Reply:
Ah, that is legacy Options data structure we are using.